icons/citigroup.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24"><path fill="#255be3" d="M0 9.868c0-2.616 2.035-4.653 4.687-4.653 1.535 0 2.936.72 3.736 1.809l-1.15 1.211a3.34 3.34 0 0 0-1.14-.976 3.16 3.16 0 0 0-1.43-.358c-1.584 0-2.852 1.246-2.852 2.967 0 1.739 1.268 2.985 2.853 2.985a3.2 3.2 0 0 0 1.48-.366c.46-.241.861-.59 1.172-1.02l1.134 1.176c-.767 1.141-2.235 1.896-3.803 1.896C2.035 14.539 0 12.502 0 9.869M9.991 5.531h1.819v8.692H9.99ZM15.196 11.888V7.147H13.21V5.53h2.068V3.67l1.701-.878v2.74h2.703v1.615H16.98v4.424c0 .878.467 1.265 1.351 1.265.46.002.916-.1 1.335-.299v1.65a3.8 3.8 0 0 1-1.668.352c-1.618 0-2.802-.93-2.802-2.651M21.117 5.531h1.818v8.692h-1.818z" style="fill:#000;stroke-width:.121562"/><path fill="#ff3c28" d="M16.446 0a8.93 8.93 0 0 1 4.293 1.092 9.5 9.5 0 0 1 3.33 3.052h-2.118a7.4 7.4 0 0 0-2.492-1.907 7.1 7.1 0 0 0-3.013-.674 7.1 7.1 0 0 0-3.012.674 7.4 7.4 0 0 0-2.492 1.907H8.824a9.5 9.5 0 0 1 3.33-3.052A8.93 8.93 0 0 1 16.446 0" style="fill:#000;stroke-width:.121562"/></svg>
```

Comment:
Please fix the problem mentioned in the Github actions (https://github.com/simple-icons/simple-icons/actions/runs/7980352457/job/21789793484?pr=10496).

You should take a look at our [contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md) since there are a few things wrong actually. To name a few: multiple paths where it should be only one, unnecessary attributes, not centered, ...

Reply:
Yeah, thanks for the feedback! On it!

Reply:
@performant23 Thanks for your work on this. There are still some issues with the icon. You will need to

- Compound all paths into one. to keep a single `path` element
  - If you're using Inkscape, you can select `union` or `combine` in the `path` menu after selecting every path element
- Center the icon vertically
  - If you're using Inkscape, you can select `Align and distribute` in the `object` menu
  - You'll be able to center the element using 
![image](https://github.com/simple-icons/simple-icons/assets/12817388/3a4f374d-0dae-4918-9745-c9f330eb0d39)
- Remove any unnecessary attributes
  - You can do this with a text editor if needed. You'll need to remove any `fill` and `style` attribute
  
  You can take a look at the steps mentioned in [section "Extract the Icon from the Logo" of our contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#2-extract-the-icon-from-the-logo). There is also some discussions that might be helpful in https://github.com/simple-icons/simple-icons/discussions/categories/help-with-svgs

Reply:
The checks should be all successful to be able to merge this PR:
![image](https://github.com/simple-icons/simple-icons/assets/12817388/190947b2-53fc-4022-b555-b5fe6a35a77b)